openc3/spec/models/target_model_spec.rb

```diff
@@ -84,6 +200,20 @@ module OpenC3
         model.update_store(System.new(['EMPTY'], File.join(SPEC_DIR, 'install', 'config', 'targets')))
       end
 
+      it "can set packet" do
+        pkts = TargetModel.packets("INST", type: :TLM, scope: "DEFAULT")
+        model = TargetModel.new(folder_name: "INST", name: "INST", scope: "DEFAULT")
+        TargetModel.set_packet('INST', 'ADCS', pkts[0], type: :TLM, scope: "DEFAULT")
+      end
+
+      it "calls limits_groups" do
+        TargetModel.limits_groups(scope: 'DEFAULT')
+      end
+
+      it "gets item-to-packet map" do
+        TargetModel.get_item_to_packet_map("INST", scope: "DEFAULT")
+      end
+
```

Comment:
None of these are testing anything ... there are no expect statements.

Reply:
Fixed 'em.